CHANGES.rst

```diff
@@ -1,6 +1,21 @@
 platformdirs Changelog
 ======================
 
+platformdirs 2.2.0
+------------------
+- [Issue 3] Add ``*_path`` API, returning :class:`pathlib.Path <pathlib.Path>` objects instead of :class:`str`
+  - by `@papr <https://github.com/papr/>`_
+
+  * **NEW** :func:`user_data_path <platformdirs.user_data_path>`
+  * **NEW** :func:`user_config_path <platformdirs.user_config_path>`
+  * **NEW** :func:`user_cache_path <platformdirs.user_cache_path>`
+  * **NEW** :func:`user_state_path <platformdirs.user_state_path>`
+  * **NEW** :func:`user_log_path <platformdirs.user_log_path>`
+  * **NEW** :func:`site_data_path <platformdirs.site_data_path>` (**Note:** Always returns first item, even if the
+    ``multipath`` argument is set to ``True``)
```

Comment:
The note here is not needed, is part of the documentation the user might read there.